ci/terraform/utils/common_passwords_update_s3.tf

```diff
@@ -0,0 +1,40 @@
+resource "aws_s3_bucket" "common_passwords" {
+  bucket = "${var.environment}-common-passwords-bucket"
+  tags   = local.default_tags
+
+  lifecycle {
+    prevent_destroy = true
+  }
+}
+
+resource "aws_s3_bucket_acl" "common_passwords" {
+  bucket = aws_s3_bucket.common_passwords.id
+  acl    = "private"
+}
+
+resource "aws_s3_bucket_versioning" "common_passwords" {
+  bucket = aws_s3_bucket.common_passwords.id
+  versioning_configuration {
+    status = "Enabled"
+  }
+}
+
```

Comment:
By default, we should probably also include server side encryption config on all our buckets: https://registry.terraform.io/providers/hashicorp%20%20/aws/latest/docs/resources/s3_bucket_server_side_encryption_configuration